docs/README.rst

```diff
@@ -0,0 +1,734 @@
+
+   .. code-block:: bash
+      :caption: Debian/Ubuntu
+
+      $ sudo service f5-oslbaasv2-agent stop
+      $ pip uninstall f5-openstack-agent
+
+   .. code-block:: bash
+      :caption: Red Hat/CentOS
+
+      $ sudo systemctl stop f5-openstack-agent
+      $ sudo systemctl disable f5-openstack-agent
+      $ sudo pip uninstall f5-openstack-agent
+
+#. Follow the :ref:`installation <agent-installation>` instructions to install a different version of the |agent-short|.
+
+#. Copy your configuration file back into :file:`/etc/neutron/services/f5`.
+
+   .. tip::
+
+      It's good practice to compare your saved copy of the configuration file with the new one created during installation.
+      Verify that the only differences between the two are those required for your deployment.
+      If new options appear in the config file, see :ref:`supported features <agent-supported-features>` and :ref:`configuration parameters <agent-config-parameters>` for explanations and config instructions.
+
+   .. code-block:: bash
+
+      $ cp ~/f5-upgrade-temp/f5-openstack-agent.ini /etc/neutron/services/f5/f5-openstack-agent.ini
+
+
+.. rubric:: Footnotes
+.. [#neutroncli] See the `Neutron LBaaS documentation <https://docs.openstack.org/mitaka/networking-guide/config-lbaas.html>`_
+.. [#agent] Similar to BIG-IP :term:`high availability`, but applies to the |agent-short| processes.
+
+.. |Build Status| image:: https://travis-ci.org/F5Networks/f5-openstack-agent.svg?branch=liberty
+   :target: https://travis-ci.org/F5Networks/f5-openstack-agent
+   :alt: Travis-CI Build Status
+.. _OpenStack provider network: https://docs.openstack.org/newton/networking-guide/intro-os-networking.html#provider-networks
+.. _Address Resolution Protocol: https://support.f5.com/kb/en-us/products/big-ip_ltm/manuals/product/tmos-routing-administration-13-0-0/11.html
+.. _Neutron Modular Layer 2: https://wiki.openstack.org/wiki/Neutron/ML2
+.. _BIG-IP route domains: https://support.f5.com/kb/en-us/products/big-ip_ltm/manuals/product/tmos-routing-administration-13-0-0/8.html
+.. _BIG-IP SNAT pools: https://support.f5.com/kb/en-us/products/big-ip_ltm/manuals/product/tmos-routing-administration-13-0-0/7.html
+.. _OpenStack Barbican: https://wiki.openstack.org/wiki/Barbican
+.. _OpenStack Keystone: https://wiki.openstack.org/wiki/Keystone
+.. _Binds VLANs to BIG-IP interfaces: https://support.f5.com/kb/en-us/products/big-ip_ltm/manuals/product/tmos-routing-administration-13-0-0/4.html
+.. _automap SNAT: https://support.f5.com/kb/en-us/products/big-ip_ltm/manuals/product/tmos-routing-administration-13-0-0/7.html
+.. _configures the requested objects: /cloud/openstack/latest/lbaas/bigip-command-mapping.html
+.. _Distributed Virtual Router: https://specs.openstack.org/openstack/neutron-specs/specs/juno/neutron-ovs-dvr.html
+.. _Role Based Access Control: http://specs.openstack.org/openstack/neutron-specs/specs/liberty/rbac-networks.html
```

Comment:
For the RBAC link (`Role Based Access Control: http://specs.openstack.org/openstack/neutron-specs/specs/liberty/rbac-networks.html`) it needs to stay liberty -- this refers to the OpenStack version the feature was introduced in.